packages/@lightningjs/ui-components/src/docs/LightningResources.mdx

```diff
@@ -0,0 +1,50 @@
+﻿{/* prettier-ignore */}
+{/*Copyright 2023 Comcast Cable Communications Management, LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+SPDX-License-Identifier: Apache-2.0\*/}
+
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Docs/Lightning Resources" />
+
+## Additional Resources
+
+This page provides additional resources that can help users learn more about Lightning-UI.
+
+### Lightning Documentation
+
+- https://lightningjs.io/docs/#/what-is-lightning/index
+
+- https://rdkcentral.github.io/Lightning/
+
+- https://rdkcentral.github.io/Lightning-SDK/
+
+### Comcast Internal Community Guide
+
+- https://github.comcast.com/Lightning/lightning-community-guide
+
+### Lightning UI Components
+
+- ##### Inner Source Repo
+  - https://github.com/rdkcentral/Lightning-UI-Components
+- ##### Open Source Repo
+  - https://github.com/rdkcentral/Lightning-UI-Components
+
+### Lightning Tutorials
+
+- ##### Video of Chelsea and Emily's Knowledgefest Talk
+  - [Building a Front-End Lightning App Video](https://web.microsoftstream.com/video/91743666-86e6-4743-b9c4-dd4b7a686a75)
+- ##### GitHub Repo for Chelsea and Emily's Knowledgefest Talk
+  - [Building a Front-End Lightning App Repo](https://github.comcast.com/XD-Engineering/knowledgefest-lightning-workshop)
+- ##### Comcast Entertainment Organization's Lightning Strategy Videos
+  - https://web.microsoftstream.com/channel/886afb0d-3401-44ee-a322-abd4676f3bc4
```

Comment:
Wow I totally forgot about these, these are all internal links so we should probably remove these and manually add on the IS LUI side 🤦‍♀️

Reply:
same with the community guide on line 34